recipes/pysnooper/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "PySnooper" %}
+{% set version = "0.0.34" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 48e8144f237fce79e3f7d1c5bf32911e3db338767eab70a9c3a75c6e5c4b543c
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - pysnooper
+    - pysnooper.third_party
+
+about:
+  home: https://github.com/cool-RR/PySnooper
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Conda automatically looks in the recipe directory if it isn’t found in the work directory:
```suggestion
  license_file: LICENSE
```